docusaurus/docs/resources/native-form.md

```diff
@@ -29,23 +29,23 @@ nativeForm(spaceId[, params[, formAttributes][, type]]);
 
 ### Required params
 
--   spaceId: String
+- spaceId: String
 
 ### Optional params
 
--   params: Object. Additional parameters you want sent in the post.
--   formAttributes: Object. Set/override the form attributes like `target`, `method`, and `action`. `method` defaults to "post", `action` will default to "\`/ms/api/availity/internal/spc/magneto/sso/v1/saml/\${spaceId}\`", and `target` will default to "\_blank". Additional attributes can be defined and should be valid on an HTML form element.
--   type: String. Override the magneto integration type. Defaults to "saml"
+- params: Object. Additional parameters you want sent in the post.
+- formAttributes: Object. Set/override the form attributes like `target`, `method`, and `action`. `method` defaults to "post", `action` will default to "\`/ms/api/availity/internal/spc/magneto/sso/v1/saml/\${spaceId}\`", and `target` will default to "\_blank". Additional attributes can be defined and should be valid on an HTML form element.
```

Comment:
`action` now defaults to `/ms/api/availity/internal/spc/magneto/sso/v1/${typeLower}/${spaceId}`
where `typeLower` is either `saml` or `openid`